requirements.txt

```
@@ -25,6 +25,11 @@ transformers~=4.28.1
 python-dateutil~=2.8.2
 nltk~=3.8.1
 rapidfuzz~=2.13.7
+atlassian-python-api
+bs4
+psutil
+requests
+PyPDF2
```

One last little tiny thing. As far as I can tell the atlassian-python-api and psutil seem to be duplicate and PyPDF2 seems to be an old added import. But other than that. very happy. Great job. If you agree you can just click commit suggestion.
```suggestion
bs4
requests
```

---

Indeed, seems like not all have been pushed and thanks for suggesting :)